### DISCUSSION TITLE
20 kHz channel step

### DISCUSSION THREAD

**Discussion by jh4vaj** [egzumer/uv-k5-firmware-custom#295] [Ideas]
Is it possible to add 20 kHz to the channel step?
In Japan, the HAM band operates at 20kHz intervals.
I would appreciate your consideration.

**Comment by egzumer** [comment]
Those steps are somebody else's idea. Strange 20kHz is not there as it seems to be known step. I'll add that but I don't want to change that ever again as it creates a configuration mess. Are there any other useful steps that are not there? Do you have any other decent radio that you can check wat steps are available there?

**Comment by egzumer** [comment]
I found Yaesu FT5DE:
5, 6.25, 8.33, 9, 10, 12.5, 15, 20, 25, 50, 1OOkHz

**Comment by jh4vaj** [comment]
ALINCO DJ-X100 (RECEIVER) has:

1k/5k/6.25k/8.33k/10k/12.5k/15k/20k/25k/30k/50k/100k/125k/200k

I am sorry, but I do not know if all of these are necessary.

**Comment by egzumer** [comment]
I'll add 9, 20, 200

**Reply by jh4vaj** [reply]
Thank you very much.

**Reply by gartnl** [reply]
as far as I know the 9 khz is only used on medium and shortwave, and mostly superseded by 10 k.

**Reply by egzumer** [reply]
Already added, i think that doesn't hurt nobody.

**Reply by jh4vaj** [reply]
I see that the 20 kHz step has been implemented in v0.21, including the spectrum analyzer mode.
Thank you very much.

Also, it seems that there are some who are attempting to receive medium waves with UV-K5 using an up-converter.

https://www.youtube.com/shorts/lPFDoi3_dZY

For these people, the 9 kHz step will be of great help.

**Reply by gartnl** [reply]
To be honest: using an upconverter for this HT is not the most sensible thing to do. Sure, it is a nice excercise but any AM radio will do better for less money.

**Comment by egzumer** [comment]
see how that works https://github.com/egzumer/uv-k5-firmware-custom/actions/runs/7250754584/artifacts/1121846814

**Reply by jh4vaj** [reply]
When I clicked on that link, firmware.zip was downloaded.
It contains firmware.bin and firmware.packed.bin.
Should I install this on UV-K5? Which one should I install?

I apologize for the inconvenience.

**Reply by egzumer** [reply]
packed

**Reply by jh4vaj** [reply]
![2023-12-19 01-29-33](https://github.com/egzumer/uv-k5-firmware-custom/assets/97027205/8eca4dbd-9539-48a3-aec3-b9e01ccd8a4a)

Excellent!
All Japanese HAM will be pleased.
It would be even better if this step could be used in spectrum analyzer mode as well.

**Comment by ErikS-web** [comment]
*** Implemented in release 0.21  **
